src/aat/resources/features/common/case/befta-jurisdiction2-case-creation_by_solicitor_1/Befta_Jurisdiction2_Default_Full_Case_Creation_Data_solicitor_1.td.json

```diff
@@ -0,0 +1,159 @@
+{
+  "_guid_": "Befta_Jurisdiction2_Default_Full_Case_Creation_Data_solicitor_1",
+  "_extends_": "Befta_Jurisdiction2_Case_Creation_Base_Data_Solicitor_1",
+  "users": {
+    "invokingUser": {
+      "username": "befta.caseworker.2.solicitor.1@gmail.com",
+      "password": "Pa55word11"
+    }
+  },
+  "request": {
+    "headers": {
+      "_extends_": "Common_Request_Headers"
+    },
+    "pathVariables": {
+      "jid": "BEFTA_JURISDICTION_2",
+      "ctid": "BEFTA_CASETYPE_2_1"
+    },
+    "body": {
+      "data": {
+        "_extends_": "Befta_Jurisdiction2_Default_Full_Case_Data_Solicitor_1"
+      }
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 201,
+    "responseMessage": "OK",
+    "headers": {
+      "_extends_": "Common_Response_Headers",
+      "Vary": "Accept-Encoding"
+
+    },
+    "body": {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": "[[ANY]]",
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
+        "DocumentField4": {
```

Comment:
Please extend this from data file. It will help in reducing the clutter across JSON.

Reply:
Need Clarity.

Reply:
Done